packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/translations/Messages_fr_FR.json

```
@@ -0,0 +1,24 @@
+{
+  "pci_databases_general_information_info_title": "Information",
+  "pci_databases_general_information_name": "Nom",
+  "pci_databases_general_information_status_label": "Status",
+  "pci_databases_general_information_nodes_label": "Nœuds",
+  "pci_databases_general_information_add_nodes": "Ajouter des nœuds",
+  "pci_databases_general_information_service_label": "Service",
+  "pci_databases_general_information_upgrade_version": "Upgrade version",
+  "pci_databases_general_information_plan_label": "Plan",
+  "pci_databases_general_information_upgrade_plan": "Upgrade range",
+  "pci_databases_general_information_region_label": "Datacentre",
+  "pci_databases_general_information_configuration_title": "Configuration",
+  "pci_databases_general_information_maintenance_window_label": "Maintenance window",
+  "pci_databases_general_information_maintenance_window": "{{start}} à {{end}}",
+  "pci_databases_general_information_private_network_label": "Private network",
+  "pci_databases_general_information_allowed_ips_label": "Adresses IP autorisées",
+  "pci_databases_general_information_manage_ips": "Gérer les adresses IP",
+  "pci_databases_general_information_service_uri_label": "Service URI",
+  "pci_databases_general_information_subscription_title": "Abonnement",
+  "pci_databases_general_information_manage_billing_label": "Gérer la facturation",
+  "pci_databases_general_information_display_billing": "Afficher la facturation",
+  "pci_databases_general_information_creation_date": "Date de création",
+  "manager_billing_subscription_next_due_date": "Prochaine échéance"
```

i think we consistent with the rest of translation keys.
```suggestion
  "pci_databases_general_information_creation_date": "Date de création",
  "pci_databases_general_information_billing_subscription_next_due_date": "Prochaine échéance"
```

---

This key is not used and has been removed